### PR TITLE
add remaining model parameter files

### PR DESCRIPTION
**Pull request by dgergel**
This PR adds yaml files for the remaining models that will be included in round 1 for tasmax.

### CODE REVIEW

File: workflows/parameters/UKESM1-0-LL-tasmax.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+jobs: |
+  [
+    {
+      "target": "historical",
+      "variable_id": "tasmax",
+      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmax", "source_id": "UKESM1-0-LL", "institution_id": "MOHC", "member_id": "r1i1p1f2", "grid_label": "gn", "version": "20190627	" },
```
Comment (brews) [inline workflows/parameters/UKESM1-0-LL-tasmax.yaml:6]: The "version" entry on this line has a bit of whitespace in the string... not sure if this is intentional?

File: workflows/parameters/UKESM1-0-LL-tasmin.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+jobs: |
+  [
+    {
+      "target": "historical",
+      "variable_id": "tasmin",
+      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "UKESM1-0-LL", "institution_id": "MOHC", "member_id": "r1i1p1f2", "grid_label": "gn", "version": "20190627	" },
```
Comment (brews) [inline workflows/parameters/UKESM1-0-LL-tasmin.yaml:6]: The "version" entry on this line has a bit of whitespace in the string... not sure if this is needed?

File: workflows/parameters/UKESM1-0-LL-pr.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+jobs: |
+  [
+    {
+      "target": "historical",
+      "variable_id": "pr",
+      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "pr", "source_id": "UKESM1-0-LL", "institution_id": "MOHC", "member_id": "r1i1p1f2", "grid_label": "gn", "version": "20190627	" },
```
Comment (brews) [inline workflows/parameters/UKESM1-0-LL-pr.yaml:6]: The "version" entry on this line has a bit of whitespace in the string... not sure if this is intentional?

**Comment by brews** [comment]
@dgergel I found extra white space in the "version" value for some of the entires. Is this required/intentional?

File: workflows/parameters/UKESM1-0-LL-tasmax.yaml (hunk 1 above)
Comment (dgergel) [reply workflows/parameters/UKESM1-0-LL-tasmax.yaml:6]: It wasn't - thanks for catching this!

File: workflows/parameters/UKESM1-0-LL-tasmin.yaml (hunk 2 above)
Comment (dgergel) [reply workflows/parameters/UKESM1-0-LL-tasmin.yaml:6]: fixed - thanks!

File: workflows/parameters/UKESM1-0-LL-pr.yaml (hunk 3 above)
Comment (dgergel) [reply workflows/parameters/UKESM1-0-LL-pr.yaml:6]: fixed

**Comment by dgergel** [comment]
@brews thanks for catching that - it was a mistake. Should be all fixed now.

**Review by brews — APPROVED** [review]
@dgergel With your last commits this LGTM. We can update later get caught in the cleaner when we actually run things.

I think this is set to merge whenever you're ready. 👍

**Pull request merged**